### DISCUSSION TITLE
🎁 Feature Wishlist

### DISCUSSION THREAD

**Discussion by Zimbelstern** [Zimbelstern/Tournant#6] [Polls]
Which features would you like to see in future versions of Tournant?

Vote and tell me your ideas!

I will consider implementing the most popular ideas – but no promises!

By the way, any features which need to modify recipes (or the recipe list), will be postponed because modifying does not make sense without bidirectional syncing to Gourmand.

**Comment by pauldausi** [comment]
Hello
I didn't manage to use tournant. I didn't understand how to find, recover these xml.
gourmet is on my computer, it's easy, the screen is big.
is it possible to open a web page and save it in tournant? thanks

**Reply by Zimbelstern** [reply]
Hi, have you read [README.md](https://github.com/zimbelstern/Tournant#readme)? Tournant can only parse recipes exported by Gourmand (Gourmet).
Assuming that you have a current version of [Gourmand](https://github.com/GourmandRecipeManager/gourmand) installed, you need to do the following steps.

0. Create some recipes in Gourmand
1. Choose `File` -> `Export all recipes` and save the file as `Gourmet XML file` (file extension `.xml` or `.grmt`).
2. Transfer the file to your Android device (how you do this is up to you).
3. Tournant should be able to open this file.

Tournant can not parse recipes from web pages or any other sources (yet). You could import the web page with Gourmand (I have not tested this) and then repeat steps 1-3.

**Comment by IWarnedYouAboutStairs** [comment]
Would you consider adding a tagging system similar to the Category or Cuisine entries that took an arbitrary number of words? So for example you could have Category: Breakfast, Cuisine: American, Tags: Fast, Sweet, Baked, etc?

**Reply by Zimbelstern** [reply]
Hi, great idea. I have some higher priority issues at the moment but I will put it on my agenda for next year.
